src/main/java/edu/kis/powp/jobs2d/drivers/macro/MacroDriverDecorator.java

```diff
@@ -40,11 +40,13 @@ public Job2dDriver getDriver() {
 	@Override
 	public void setPosition(int x, int y) {
 		driverCommandList.add(new SetPositionCommand(x, y));
+		driver.operateTo(x,y);
```

Comment:
What happened here?! The lines added here are not related to your task? Where did you get those (copy&paste?)?!

Reply:
This was to solve that "Bug: drawing does not work during recording MACRO" and it slipped with one commit and i didn't notice it go now but it is now drawing lines during recording macro. So that's what happend.

Reply:
OK, but it would be nice to have it in a separate PR (but this require working on different branches on different features)